runtime/service/npud/core/DevManager.cc

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "DevManager.h"
+
+#include <util/ConfigSource.h>
+
+namespace npud
+{
+namespace core
+{
+
+#define DEFAULT_DEVICE_PATH "/usr/lib/npud/devices"
```

Comment:
Is this line necessary?

Reply:
Sure. The backend library will be located here.

Reply:
I don't understand well. This definition is not used anywhere and `DevManager` uses `util::config::DEVICE_MODULE_PATH`. Are they different?